frontend/app/src/main/java/com/quickpick/payloads/LoginPayload.java

```diff
@@ -0,0 +1,13 @@
+package com.quickpick.payloads;
+
+public class LoginPayload {
+
+    private String facebookToken;
+
```

Comment:
rm whitespace

Reply:
Addressed, we'll use newlines between members unless they make sense to be grouped.